docs/old/Plugins.md

```diff
@@ -45,13 +45,13 @@ from aac.parser import parse_file, parse_str
 from aac.validator import validation
 
 def my_plugin_command(architecture_file: str):
-    with validation(parse_file, architecture_file) as model:
-        if not model:
+    with validation(parse_file, architecture_file) as result:
+        if result.is_valid:
```

Comment:
Is this not checking for an invalid result?
```suggestion
        if not result.is_valid:
```

Reply:
Good catch! This is actually out-dated, as written. Updated in commit d3d79abf99297a454c6e78398d5373f7f55de1bb